internal/cmd/cloud_login.go

```diff
@@ -92,18 +101,47 @@ func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
 	show := getNullBool(cmd.Flags(), "show")
 	reset := getNullBool(cmd.Flags(), "reset")
 	token := getNullString(cmd.Flags(), "token")
+	stackSlug := getNullString(cmd.Flags(), "stack-slug")
```

Comment:
Is the pull request's description outdated, or they are intentionally different?